public/js/icinga/loader.js

```diff
@@ -647,19 +647,23 @@
             if (!! extraUpdates && req.getResponseHeader('X-Icinga-Redirect-Http') !== 'yes') {
                 var _this = this;
                 $.each(extraUpdates.split(','), function (idx, el) {
-                    var parts = el.trim().split(';');
-                    if (parts.length !== 2) {
+                    var parts = el.trim().split(';'), $target, url;
```

Comment:
Please don't initialize variables on a single line. Use a separate line and `var` for each one.